arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -85,15 +85,16 @@ class EditAndSyncFeaturesViewController: UIViewController {
             if let error = error {
                 self.presentAlert(error: error)
             } else {
-                if let featureServiceInfo = self.geodatabaseSyncTask?.featureServiceInfo,
-                    let map = self.mapView.map {
-                    for index in featureServiceInfo.layerInfos.indices.reversed() {
```

Comment:
Does it make a difference to remove `reversed()`?

Reply:
It doesn't make a difference since we're only showing the point feature layer. Previously, it prevented the polygon layer from being added on top of the point layer which caused issues when trying to select a point.